lib/ReactViews/WelcomeMessage/WelcomeMessage.jsx

```diff
@@ -1,75 +1,119 @@
 import React, { useState } from "react";
-import createReactClass from "create-react-class";
+import { withTranslation } from "react-i18next";
+import { withTheme } from "styled-components";
 import PropTypes from "prop-types";
-import classNames from "classnames";
 
-import knockout from "terriajs-cesium/Source/ThirdParty/knockout";
-
-import Styles from "./welcome-message.scss";
-import Icon from "../Icon.jsx";
+import Icon, { StyledIcon } from "../Icon.jsx";
 import FadeIn from "../Transitions/FadeIn/FadeIn";
 import SlideUpFadeIn from "../Transitions/SlideUpFadeIn/SlideUpFadeIn";
 
 import Spacing from "../../Styled/Spacing";
+import Box from "../../Styled/Box";
+import Text, { TextSpan } from "../../Styled/Text";
 
 import { useKeyPress } from "../Hooks/useKeyPress.js";
-import { useTranslation, Trans } from "react-i18next";
-import { runInAction } from "mobx";
+// import { useTranslation, Trans } from "react-i18next";
+import { observer } from "mobx-react";
+import styled from "styled-components";
+import Button, { RawButton } from "../../Styled/Button";
+import VideoGuide from "../Map/Panels/HelpPanel/VideoGuide";
 
 export const WELCOME_MESSAGE_NAME = "welcomeMessage";
 export const LOCAL_PROPERTY_KEY = `${WELCOME_MESSAGE_NAME}Prompted`;
+const WELCOME_MESSAGE_VIDEO = "welcomeMessageVideo";
+
+const WelcomeModalWrapper = styled(Box)`
+  z-index: 99999;
+  background-color: rgba(0, 0, 0, 0.75);
+`;
+
+function WelcomeMessageButton(props) {
+  return (
+    <Button
+      primary
+      rounded
+      fullWidth
+      css={`
+        height: 50px;
```

Comment:
The design shows it using the default 40px height buttons, is there a reason why we need 50 here? Also related: https://github.com/TerriaJS/terriajs/pull/4293/files#r424149884

Reply:
Seems to be a sketch vs invision thing, on invision inspect it says 50px 😬 I'll remove it then if it says 40px on sketch

Reply:
Ah you're right - invision hasn't been updated. use 40px (and @vicborgy if there's any other differences you know between what you had on invision and your latest onboarding, please note them for regina :) )